content/Simple dynamic settings for Django - Jorge Michelena.md

```diff
@@ -0,0 +1,101 @@
+## Simple dynamic settings for Django
+
+
+### Introduction
+
+Not long ago I finished working on a small project in Django when I realized that it could be improved greatly by adding a single feature.
```

Comment:
```suggestion
I recently finished working on a small project in Django when I realized that it could be improved greatly by adding a single feature.
```